partiql-conformance-test-generator/src/schema.rs

```diff
@@ -0,0 +1,42 @@
+/// Conformance test document containing namespaces and/or tests
```

Comment:
Non-blocking for this PR: Does having a TODO that references the test ISL and later on schema code generation based on that makes sense here?

Reply:
I can add a TODO linking to the test ISL once it's added to `partiql-tests`. Also will add a TODO for schema code generation based on ISL.